ignite/metrics/accuracy.py

```diff
@@ -14,9 +14,11 @@ def __init__(
         self,
         output_transform: Callable = lambda x: x,
         is_multilabel: bool = False,
+        is_variable_multiclass: bool = False,
         device: Optional[Union[str, torch.device]] = None,
     ):
         self._is_multilabel = is_multilabel
+        self._is_variable_multiclass = is_variable_multiclass
```

Comment:
We need to check that this flag is False in case of multi-label

Reply:
done!